cl_manycore/hardware/Makefile.include

```diff
@@ -28,7 +28,7 @@ $(HARDWARE_PATH)/bsg_bladerunner_configuration.rom: $(CL_DIR)/Makefile.dimension
                 bsg_f1@$(BSG_F1_COMMIT_ID) >> $@
 
 $(HARDWARE_PATH)/%.v: $(HARDWARE_PATH)/%.rom
-	python $(BSG_IP_CORES_DIR)/bsg_mem/bsg_ascii_to_rom.py $< \
+	python $(BASEJUMP_STL_DIR)/bsg_mem/bsg_ascii_to_rom.py $< \
```

Comment:
Are we going with BASEJUMP_STL_DIR instead of BSG_IP_CORES_DIR in all repositories from now on (including bsg_manycore) or is it just for F1?

Reply:
I switched, but both environment variables still are set

Reply:
But it won't work if you have BSG_IP_CORES_DIR set and BASEJUMP_STL_DIR not set right? if that's the case people should know about it, I'll add it to the environment configuration in the technical reference manual, are there any other places we should add it to? READMEs and such?

Reply:
Do you want me to switch back to BSG_IP_CORES_DIR?

Reply:
If you run in bladerunner (which is the recommended method) it sets both of these variables

Reply:
Oh no no it's about time we used BASEJUMP_STL_DIR anyway, I just wanted to make sure people don't get blindsided. If bladerunner sets it them I'm fine with it!